vaxis.go

```diff
@@ -1571,6 +1571,11 @@ func (vx *Vaxis) SetTitle(s string) {
 	_, _ = io.WriteString(vx.console, tparm(setTitle, s))
 }
 
+// SetPath sets the terminal's working directory via OSC 7
+func (vx *Vaxis) SetWorkDir(s string) {
+	_, _ = io.WriteString(vx.console, tparm(setCWD, s))
```

Comment:
Can you change this to `NotifyWorkingDirectory`? I think that conveys what is happening better (IE we are notifying the terminal that we have a new working directory).